unittests/CMakeLists.txt

```diff
@@ -6,10 +6,15 @@ add_subdirectory(POSIX/)
 add_subdirectory(gvisor-tests/)
 add_subdirectory(gcc-target-tests-32/)
 add_subdirectory(gcc-target-tests-64/)
+
 if (BUILD_THUNKS)
   add_subdirectory(ThunkLibs)
 endif()
 
 if (BUILD_FEX_LINUX_TESTS)
   add_subdirectory(FEXLinuxTests/)
 endif()
+
+if (BUILD_THUNKS)
+  add_subdirectory(ThunkFunctionalTests)
```

Comment:
Why not move this into the same scope as ThunkLibs for grouping?

Reply:
Resolved.